src/base-handler.ts

```diff
@@ -121,16 +129,21 @@ export abstract class BaseProxyHandler {
         return getPrototypeOf(originalTarget);
     }
     getOwnPropertyDescriptor(shadowTarget: ReactiveMembraneShadowTarget, key: PropertyKey): PropertyDescriptor | undefined {
-        const { originalTarget, membrane: { valueObserved } } = this;
+        const { originalTarget, membrane: { valueObserved, tagPropertyKey } } = this;
 
         // keys looked up via getOwnPropertyDescriptor need to be reactive
         valueObserved(originalTarget, key);
 
-        const desc = getOwnPropertyDescriptor(originalTarget, key);
+        let desc = getOwnPropertyDescriptor(originalTarget, key);
         if (isUndefined(desc)) {
-            return undefined;
+            if (key !== tagPropertyKey) {
+                return undefined;
+            }
+            // if the key is the membrane tag key, and is not in the original target,
+            // we produce a synthetic descriptor and install it on the shadow target
+            desc = { value: undefined, writable: false, configurable: false, enumerable: false };
+            ObjectDefineProperty(shadowTarget, tagPropertyKey, desc);
```

Comment:
It is safe to return the `desc` at this point. This one is pretty harmless and does not need to be wrapped.